packages/block-editor/src/components/global-styles/filters-panel.js

```diff
@@ -75,6 +90,36 @@ const DEFAULT_CONTROLS = {
 	duotone: true,
 };
 
+const popoverProps = {
+	placement: 'left-start',
+	offset: 36,
+	shift: true,
+	className: 'block-editor-duotone-control__popover',
+	headerTitle: __( 'Duotone' ),
+};
+
+const LabeledColorIndicators = ( { indicators, label } ) => (
+	<HStack justify="flex-start">
+		<ZStack isLayered={ false } offset={ -8 }>
+			{ indicators.map( ( indicator, index ) => (
+				<Flex key={ index } expanded={ false }>
+					{ indicator === 'unset' || ! indicator ? (
+						<ColorIndicator className="block-editor-duotone-control__unset-indicator" />
```

Comment:
I saw this check and this class in a couple places, It seems to me that the `DuotonePicker` already understands the `unset` value, so it's not far fetched to say that the `DuotoneSwatch` should also understand the `unset` value and do the right thing internally instead of duplicating the styles and the classname every time.